dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -126,17 +126,18 @@ internal class SdkFeature(
 
     // region FeatureScope
 
-    override fun withWriteContext(callback: (DatadogContext, EventBatchWriter) -> Unit) {
+    override fun withWriteContext(
+        forceNewBoolean: Boolean,
```

Comment:
`forceNewBatch`, maybe? :)

```suggestion
        forceNewBatch: Boolean,
```

Reply:
Damn...yes, miss typing